### DISCUSSION TITLE
Adding crop and basic editing option for image block

### DISCUSSION THREAD

**Discussion by swapnilxi** [GitbookIO/community#190] [Feature Requests]
### What's your feature/idea?

when we add image there is no option and do basic editing, it would be great if I can do inside the gitbook

### What's your desired outcome?

Basic editing cropping and resizing images for better visiblity

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

**Comment by aikoii** [comment]
Hi @swapnilxi,

Thanks for your feedback!

Although we don't have the option to crop images just yet, I'm happy to let you know that we've released a feature for image resizing 🎉.

You can read our [documentation](https://docs.gitbook.com/content-creation/blocks/insert-images#resizing) for more details.

**Reply by swapnilxi** [reply]
great, but cropping comes in handy if we take full page screenshots and want something specific, or highlight something specific

**Reply by aikoii** [reply]
I definitely understand how this feature could be helpful 🙂, and I've captured this for our product team. I cannot guarantee this will be changed in the future, but we will keep you informed.

**Reply by swapnilxi** [reply]
sure, looking for a positive outcome and a better product.
